benchmark/src/lib.rs

```diff
@@ -342,62 +352,107 @@ impl Benchmarker {
             .expect("Failed to query TXN status from debug interface")
     }
 
-    /// Wait for #committed TXNs reaching target with fixed number of query iterations.
-    fn wait_for_commit(&self, target: i64) -> u128 {
-        let mut max_iterations = MAX_WAIT_COMMIT_ITERATIONS;
+    /// Wait for accepted TXNs to commit or time out: for any account, if its sequence number
+    /// (bumpped during TXN generation) equals the one synchronized from validator,
+    /// denoted as sync sequence number, then all its TXNs are committed.
+    /// Return senders' most up-to-date sync sequence numbers and how long we have waited.
+    pub fn wait_txns(&self, senders: &[AccountData]) -> (HashMap<AccountAddress, u64>, u128) {
```

Comment:
Reply to Sonia
> wait_for_account_committed_txns

I don't know, this name looks to me very complicated. But maybe that's because I am still learning English. I just feel `wait_txns` and `submit_txns` are a good pair of names.